### ISSUE TITLE
External Sharing Setup for a Site

### ISSUE THREAD

**Issue by MurtazK** [MicrosoftDocs/OfficeDocs-SharePoint#2422]
SHARING SETTINGS AT ORG-LEVEL
- External sharing = New and existing guest
- File and folder links = Specific people
- Permission for sharing links = Edit
- All options under Other settings are checked

SHARING FOR A TEST SITE (**Custom**)
- External sharing = Existing guest only
- Default sharing link type = People with existing access
- Default link permission = Same as org-level
- Limit sharing by domain unchecked
This setup functions not with guests already in the organization.
When sharing a file with an existing guest, the error we get is "The recipient you entered doesn't have permission to access the item"

What may be the possible cause?
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3106844b-caae-38c5-3c88-fb17a9d67536
* Version Independent ID: 3e88e60e-24a3-cd83-9bfc-8bc3037a86ce
* Content: [Change the external sharing setting for a site - SharePoint in Microsoft 365](https://docs.microsoft.com/en-us/sharepoint/change-external-sharing-site)
* Content Source: [SharePoint/SharePointOnline/change-external-sharing-site.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointOnline/change-external-sharing-site.md)
* Service: **sharepoint-online**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

**Comment by jensotto** [comment]
A few clarifying points:
It's important to note that the guest user has been added as a member in the Team before trying to share items with the guest. The guest therefore already has access to the SharePoint content.
The org-level setting for external sharing is set to "Existing guest only". The site external sharing has not been modified and has the same setting as the org-level defines ("Existing guest only").

This behavior makes it impossible to send a sharing link to a guest that already has access to the site. It works only for regular users in the tenant (that already has access to the site).

**Comment by jensotto** [comment]
Here is an example of what happens. The recipient is a guest and already is a member of the Team (and SharePoint site)
![image](https://user-images.githubusercontent.com/13898671/98372946-cfb8fb80-203e-11eb-82ef-86521ddaf3ee.png)

**Comment by MurtazK** [comment]
Any updates on this reported issue?

**Comment by MurtazK** [comment]
@scanum any suggestions you may want to share?

**Comment by get-itips** [comment]
Hello @jensotto Jens, how are you? Hope you are doing good.
First of all, sorry for the late response.
As here we are limited to documentation changes and improvements, I tested this on my lab to check if we needed to update any article, with an already guest-invited user, I could successfully send a sharing link to the user when selecting "Specific people", the behavior was slightly different if the destination e-mail was not an already invited user.
Hope this information helps you,
